src/gui/FileBrowser.cpp

```diff
@@ -624,42 +625,57 @@ void FileBrowserTreeWidget::focusOutEvent(QFocusEvent* fe)
 	QTreeWidget::focusOutEvent(fe);
 }
 
+void FileBrowserTreeWidget::contextMenuEvent(QContextMenuEvent* e)
+{
+#ifdef LMMS_BUILD_APPLE
+	QString fileManager = tr("Finder");
+#elif defined(LMMS_BUILD_WIN32)
+	QString fileManager = tr("Explorer");
+#else
+	QString fileManager = tr("file manager");
+#endif
```

Comment:
We should move the platform-specific code all into `FileManagerServices` and contain it there. The class might need some refactoring though.

Reply:
I can't think of a way to move this into the utility that that is cleaner and more intuitive.  Any correlation of e.g. `open --> Finder`, `explorer --> Explorer`, `xdg-open --> nautilus` feels like overkill.  This recommendation should be actionable.  As it stands, the utility class in not a better home IMO.